java/ql/src/semmle/code/java/frameworks/Jackson.qll

```diff
@@ -0,0 +1,177 @@
+/**
+ * Provides classes and predicates for working with the Jackson serialization framework.
+ */
+
+import java
+import semmle.code.java.Reflection
+import semmle.code.java.dataflow.FlowSources
+import semmle.code.java.dataflow.TaintTracking2
```

Comment:
It looks like the only dataflow-related uses in this file are references to `DataFlow::Node`.
```suggestion
private import semmle.code.java.dataflow.DataFlow
```

Reply:
This results in compilation errors in `UnsafeDeserializationQuery.qll`.

Reply:
Adding `private import semmle.code.java.dataflow.TaintTracking2` in `UnsafeDeserializationQuery.qll` helps.